bcl2fastq/index_summary.py

```diff
@@ -15,10 +15,9 @@
     './data/bcl2fastq_cache.hd5'
 )
 
-all_runs = index['Run'].sort_values(ascending=False).unique()
-
-all_runs = [{'label': r, 'value': r} for r in all_runs]
-
+all_runsog = index['Run'].sort_values(ascending=False).unique()
+print (all_runsog)
+all_runs = [{'label': r, 'value': r} for r in all_runsog]
```

Comment:
Is it possible to avoid having two global variables (`all runs` and `all_runsog`) with very similar names and content? If not, please use clearer names and comments explaining the purpose of each. Global variables can be used anywhere, so their names should be very clear.